api/writeAPIBlocking.go

```diff
@@ -74,7 +74,7 @@ func NewWriteAPIBlocking(org string, bucket string, service http2.Service, write
 func (w *writeAPIBlocking) write(ctx context.Context, line string) error {
 	err := w.service.WriteBatch(ctx, iwrite.NewBatch(line, w.writeOptions.RetryInterval(), w.writeOptions.MaxRetryTime()))
 	if err != nil {
-		return err.Unwrap()
+		return err
 	}
 	return nil
```

Comment:
Why not then just:
```go
return w.service.WriteBatch(ctx, iwrite.NewBatch(line, w.writeOptions.RetryInterval(), w.writeOptions.MaxRetryTime()))
```

Reply:
Because the return value from that is type `*http2.Error`.  If its value is `nil`, then converting it to the `error` interface produces a value that has a bound type of `*http2.Error`, but nil pointer.  That interface value does not compare equal to `nil` in the caller, making it appear an error occurred when it didn't.

(Also, because I tried that, and the tests failed due to the above problem.)

Reply:
Yes, yes, I realized that at the same time as you replied. Sorry.